### DISCUSSION TITLE
Asker ne kadar yol katetmiştir sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#226] [Zeka Sorusu]
4 km uzunluğunda bir ordu hedefe doğru ilerlemektedir. En arkada bulunan bir asker koşarak gelip en öndeki komutana bir mesaj verir ve sonra tekrar en arkadaki yerine geri döner. bu işlem sırasında ordu tam 4 kmlik bir ilerleme kaydettiğine göre askerimiz toplam ne kadar yol katetmiştir?

**Comment by YagizEfeZ** [comment]
Asker, bir turda tam 4 kmlik ilerleme yapar, ancak geri dönüşte de aynı mesafeyi geri koşarak gelir. Yani asker, toplamda 4 kmlik ilerleme kaydederek bir tur atar.

Bir turda toplam 4 km koşan asker, geri dönüşte de aynı mesafeyi geri gelir. Dolayısıyla toplam kat ettiği mesafe, bir tur ilerleme mesafesi ile geri dönüş mesafesinin toplamıdır:

Toplam Mesafe = 4 km (İlerleme) + 4 km (Geri dönüş) = 8 km

Askerimiz toplamda 8 kmlik yol katetmiştir.

**Reply by sonsuzus** [reply]
Askerin uzayda, yani x ,y,z düzleminde (tabi burada sadece yatay hareket ediyor) ne kadar yol kat ettiği soruluyor.

Gerçi düşününce toplam mesafe senin dediğin gibi de olabilir ama ordunun da hareket ediyor oluşunu hesaba kattın mı? Yani geri dönerken ordu da ona doğru geldiği için daha kısa mesafe kat eder.
